services/src/main/java/org/exoplatform/task/rest/TaskRestService.java

```diff
@@ -662,14 +664,15 @@ public Response addTaskComment(@ApiParam(value = "Comment text", required = true
   }
 
   @POST
-  @Path("comments/{commentId}/{id}")
+  @Path("comments/{commentId}/{id}/{lang}")
   @RolesAllowed("users")
   @ApiOperation(value = "Adds a sub comment to a specific parent comment by id and a specific task by id", httpMethod = "POST", response = Response.class, notes = "This Adds sub comment to a parent comment in specific task by id")
   @ApiResponses(value = { @ApiResponse(code = 200, message = "Request fulfilled"),
       @ApiResponse(code = 400, message = "Invalid query input"), @ApiResponse(code = 403, message = "Unauthorized operation"),
       @ApiResponse(code = 404, message = "Resource not found") })
   public Response addTaskSubComment(@ApiParam(value = "Comment text", required = true) String commentText,
                                     @ApiParam(value = "Comment id", required = true) @PathParam("commentId") long commentId,
+                                    @ApiParam(value = "Lang", required = true) @PathParam("lang") String lang,
```

Comment:
here you have the current user so you can have his language. No need to put it in parameter